namespaces/mson-namespace.md

```diff
@@ -583,7 +587,7 @@ Description is here! Properties to follow.
 ["object", {}, {}, [
   ["member", {}, {}, {
     "key": ["Relation", {}, {"variable": true}, "rel"],
-    "value": ["string", {}, {"typeAttributes": ["sample"]}, null]
+    "value": ["string", {}, {}, null]
```

Comment:
@klokane no, this was a mistake in the original example in the first place – `*rel (Relation)*` says nothing about what the value is so it should be the default (string)